renpy/lint.py

```diff
@@ -90,17 +95,19 @@ def try_eval(where, expr, additional=None):
     if m.group(1) in __builtins__:
         return
 
-    report( "Could not evaluate '%s', in %s.", expr, where)
+    report("Could not evaluate '%s', in %s.", expr, where)
     if additional:
         add(additional)
 
 # Returns True of the expression can be compiled as python, False
 # otherwise.
+
+
 def try_compile(where, expr, additional=None):
 
     try:
         renpy.python.py_compile_eval_bytecode(expr)
-    except:
+    except SyntaxError:
```

Comment:
Added this because my right eye twitches when try/except blocks don't specify what they're catching. However, I realized I'm here to fix print statements, not tweak exception handling. ...Then I forget to undo this before moving on.

I can either revert it, or if this function should only throw an exception on a SyntaxError, I guess we can keep it.